src/ru/skypro/Main.java

```diff
@@ -1,8 +1,99 @@
 package ru.skypro;
 
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        ///zadacha1+zadacha2
+        ///massive1
+        int[] weight = new int[3];
+        String separator = ", ";
+        weight[0] = 1;
+        weight[1] = 2;
+        weight[2] = 3;
+        for (int i = 0; i < weight.length; i++) {
+            if (i==weight.length-1) {
+                separator = "";
+            }
+            System.out.print(weight[i] + separator);
+        }
+        System.out.println();
+        ///massive2
+        String b = " ,";
```

Comment:
вы можете просто переиспользовать уже ранее определенную переменную
```suggestion
        separator = " ,";
```